app/src/main/res/drawable/ic_tracking_protection_anim_enable.xml

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
-   - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<animated-vector
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        android:drawable="@drawable/ic_tracking_protection_disabled">
-    <target android:name="strike_thru_path" android:animation="@animator/strike_thru_path_enable" />
```

Comment:
I think we can also remove

- strike_thru_path_enable
- strike_thru_mask_enable

since the `ic_tracking_protection_anim_X` were the only consumers fo this.

Reply:
Would you mind pointing where you are seeing ?

* strike_thru_path_enable
* strike_thru_mask_enable

Reply:
This ones are being used in other files

![image](https://user-images.githubusercontent.com/773158/116289043-1cfd6680-a760-11eb-87da-0710ecfbc337.png)

![image](https://user-images.githubusercontent.com/773158/116289182-43bb9d00-a760-11eb-9170-8f708d36f0e1.png)